README.md

```diff
@@ -31,69 +40,22 @@ Install dependencies with, for example:
 ```sh
 python3.10 -m pip install -r requirements.txt
 ```
+### icommands (optional)
+If the icommands are installed, the users can choose them as backend for up and downloads.
 
-## Configuration
-
-### iRODS environment.json
-
-- Please create a directory/folder named `.irods` in your home directory/folder (`~/.irods/` in Linux shorthand).
-  - Linux: `/home/\<username\>/.irods/irods_environment.json`
-  - Mac: `/Users/\<username\>/.irods/irods_environment.json`
-  - Windows: `C:\\\\....\\\<username\>\\.irods\\irods_environment.json`
-
-- Your *iRODS* admin will provide an `irods_environment.json` file, its contents, or instructions on how to create it.  Place that file into the `.irods` directory/folder.  Here it an example that can be created with the `iinit` iCommand on Linux:
-
-```json
-{
-    "irods_host": "server.fqdn.nl",
-    "irods_port": 1247,
-    "irods_user_name": "username",
-    "irods_zone_name": "myZone",
-    "irods_default_resource": "myResc"
-}
+## Usage
 ```
-
-### iBridges config.json
-
-If it does not exist, *iBridges* will create its own configuration file in `~/.ibridges/` containing the name of the last *iRODS* environment file used and other mandatory options.  This `ibridges_config.json` file can be updated to control other aspects of *iBridges*.  For example:
-
-```json
-{
-    "check_free_space": true,
-    "davrods_server": "https://server.fqdn.nl",
-    "force_transfers": false,
-    "last_ienv": "irods_environment.json",
-    "ui_tabs": [
-        "tabUpDownload",
-        "tabELNData",
-        "tabDataBundle",
-        "tabCreateTicket"
-    ]
-    "verbose": "info"
-}
+export PYTHON_IRODSCLIENT_DEFAULT_XML=QUASI_XML
```

Comment:
This variable is set within the iBridges.py script.  Will that work or will the user need to do it this way?

By the way, a Linux user can also add the variable this way:

```bash
...]$ env PYTHON_IRODSCLIENT_DEFAULT_XML=QUASI_XML ./iBridges.py
```

Reply:
I tried it without and I run more often into Segmentation Faults ... so just to be sure I would leave it in.

Reply:
Hi @chStaiger - Where do you see segfaults? In iRODS or somewhere else?

Reply:
The segfaults are caused by some test .xml files in our iRODS/YODA instance and I can prevent them with the `QUASI_XML` settings. In vanilla iRODS instances I have not encountered the segfaults.

Reply:
Okay. If the segfaults are observed in the iRODS server, please file an issue in the irods/irods repository with relevant stacktraces and logs, if possible, so that we can have a look (as it is a serious issue which we need to address). Thanks!